src/convert.ts

```diff
@@ -35,47 +147,92 @@ export function toTs(node: Flow | TSType): TSType {
     case 'TSUndefinedKeyword':
     case 'TSUnionType':
     case 'TSVoidKeyword':
+    // @ts-ignore
+    case 'TSTypeAnnotation':
+    // @ts-ignore
     case 'TSTypeParameterDeclaration':
+    // @ts-ignore
     case 'TSAsExpression':
+    // @ts-ignore
     case 'TSPropertySignature':
       return node
 
+    case 'TypeAnnotation':
+      return tsTypeAnnotation(toTsType(node))
+
+    case 'InterfaceDeclaration':
+      const { properties, spreads } = objectTypeAnnotationPropertiesAndSpreads(
+        node.body
+      )
+      if (spreads.length) {
+        throw new Error('Spreads in interfaces unsupported')
+      }
+      return tsInterfaceDeclaration(
+        node.id,
+        node.typeParameters
+          ? tsTypeParameterDeclaration(
+              node.typeParameters.params.map(toTsTypeParameter)
+            )
+          : null,
+        node.extends && node.extends.length
+          ? node.extends.map(
+              (_: InterfaceExtends): TSExpressionWithTypeArguments => toTs(_)
+            )
+          : null,
+        tsInterfaceBody(properties)
+      )
+
     // Flow types
     case 'AnyTypeAnnotation':
     case 'ArrayTypeAnnotation':
     case 'BooleanTypeAnnotation':
     case 'BooleanLiteralTypeAnnotation':
+    case 'ExistsTypeAnnotation':
     case 'FunctionTypeAnnotation':
     case 'GenericTypeAnnotation':
     case 'IntersectionTypeAnnotation':
     case 'MixedTypeAnnotation':
     case 'NullableTypeAnnotation':
     case 'NullLiteralTypeAnnotation':
-    case 'NumericLiteralTypeAnnotation':
     case 'NumberTypeAnnotation':
     case 'StringLiteralTypeAnnotation':
     case 'StringTypeAnnotation':
     case 'ThisTypeAnnotation':
     case 'TupleTypeAnnotation':
     case 'TypeofTypeAnnotation':
-    case 'TypeAnnotation':
     case 'ObjectTypeAnnotation':
     case 'UnionTypeAnnotation':
     case 'VoidTypeAnnotation':
+    // @ts-ignore
+    case 'NumericLiteralTypeAnnotation':
```

Comment:
should this be `NumberLiteralTypeAnnotation`? `Number` instead of `Numeric`. Seems to fix my error at least

Reply:
Probably; I upgraded everything since this was written so many things have been renamed and everything marked `// @ts-ignore` needs changing to the new versions.